### PR TITLE
Server metadata - Get more information from Lizmap plugin & display them in the admin panel

### PR DESCRIPTION
**Pull request by mdouchin**
Test if the Lizmap plugin for QGIS Server is installed and if QGIS Server version >= 3.16 and Lizmap plugin >= 3.7.0. If so, add more information about QGIS Server installation: version, plugins installed:

* in a new submenu of the Administration panel
2021-12-15_114935_866x446
or
2021-12-15_111616_875x560

* in the JSON returned by the URL `index.php/view/app/metadata`
```json
{
    "info": {
        "version": "3.6.0-pre",
        "date": "2021-06-21"
    },
    "dependencies": {
        "jelix": {
            "minversion": "1.1.6",
            "maxversion": "1.6.*"
        }
    },
    "qgis_server": {
        "test": "OK",
        "mime_type": "text\/xml; charset=utf-8",
        "http_code": 500,
        "response": "<ServerException>Project file error. For OWS services: please provide a SERVICE and a MAP parameter pointing to a valid QGIS project file<\/ServerException>\n",
        "metadata": {
            "version": "3.16.14",
            "name": "Hannover",
            "version_int": 31614
        },
        "support_custom_headers": null,
        "services": [
            "WMS",
            "WFS",
            "WCS",
            "WMTS",
            "ATLAS",
            "CADASTRE",
            "EXPRESSION",
            "LIZMAP"
        ],
        "plugins": {
            "wfsOutputExtension": {
                "version": "1.5.0"
            },
            "cadastre": {
                "version": "master"
            },
            "lizmap": {
                "version": "master"
            },
            "atlasprint": {
                "version": "3.0.0"
            }
        }
    }
}
```

### CODE REVIEW

**Review by Gustry — APPROVED** [review]
I love it !

**Review by laurentj — APPROVED** [review]
LGTM, thank you :)

**Review by Gustry — CHANGES_REQUESTED** [review]
I'm checking on the server side 😊

File: lizmap/modules/admin/locales/en_US/admin.UTF-8.properties [hunk 1]
```diff
@@ -223,3 +227,21 @@ upload.image.error.file.onDisk=Failed to write file to disk
...
+server.information.qgis.version=Number
+server.information.qgis.name=Name
+
+server.information.qgis.plugins=Plugins
+server.information.qgis.plugins.version=Version number
+
+server.information.qgis.test.ok=QGIS Server is correctly installed and returns the expected response.
+server.information.qgis.test.error=QGIS Server is not correctly installed in your server or the URL given in Lizmap configuration is not correct.
+server.information.qgis.error.fetching.information=We cannot get the details about your QGIS Server installation (version, plugins, etc.).
+server.information.qgis.error.fetching.information.detail=Either the version of the installed QGIS Server is under 3.16, or the Lizmap plugin is not detected in your server, or is installed with a version under 3.7.0. Please upgrade QGIS Server to minimum 3.16 and install or upgrade the Lizmap plugin to minimum 3.7.0.
```
Comment (Gustry) [inline lizmap/modules/admin/locales/en_US/admin.UTF-8.properties:247]:
```suggestion
server.information.qgis.error.fetching.information.detail=Either the version of the installed QGIS Server is under 3.10, or the Lizmap plugin is not detected in your server, or is installed with a version under 3.7.0. Please upgrade QGIS Server to minimum 3.10 and install or upgrade the Lizmap plugin to minimum 3.7.0 by reading the documentation about the environment variable https://docs.lizmap.com/3.5/en/install/pre_requirements.html#lizmap-server-plugin
```

**Review by Gustry — APPROVED** [review]
Ok for me.
The server side is ready and documentation https://docs.lizmap.com/3.5/en/install/pre_requirements.html#lizmap-qgis-server-plugin how to enable this new setting.

**Pull request merged**